### ISSUE TITLE
Incorrect angular dependency version

### ISSUE THREAD

**Issue by obibring** [angular-ui/bootstrap-bower#58]
The `bower.json` file states the angular dependency as `>=1.3.0`. This is wrong, as angular v1.4 is not yet supported.

**Comment by Foxandxss** [comment]
We will fix this for the next version where hopefully, we will have support for it.

**Comment by nschloe** [comment]
angular-bootstrap's backlog is huge, and in fact it's not even fully compatible with Angular 1.2, let alone 1.3 or 1.4 (see https://github.com/angular-ui/bootstrap/labels). For many people though, it _does_ work since they are not using carousel or whatever is broken. To let them use angular-bootstrap with bower, not restricting the version dependency makes some sense. That's of course a matter of opinion.

**Comment by Foxandxss** [comment]
Yeah, we just released 0.13.1 and I completely forgot about the bower restriction.

PS: You people should move to npm :P
